manuscript/chapter3-contributor-mode.md

```diff
@@ -243,8 +295,26 @@ We saw that the first step is to teach to others but there also other steps or t
 * Read a lot of articles (every day or every few days): use Reddit, Dev.To, newsletters like Changelog/SoftwareleadWeekly
 * Study the evening before to sleep is very powerful because helps you to remember what you read more easily
 * Create a list of stuff to read or study when you have time
+* Read stuff from others whom you admire
 * Find a way to experiment what you are learning, check the article [Don't Learn More, Learn Smarter. A Quick Guide to Agile Learning](https://www.entrepreneur.com/article/341618).
 
+I want to quote a phrase from this article [Interview candidates with an Open Source background](https://toscalix.com/2019/12/10/interview-candidates-with-an-open-source-background/) that I liked a lot:
+
+    Professionals with a solid Open Source background do not depend on certifications paid by their companies to learn how to use their tools, or having to change their base tool-set regularly, limited by commercial decisions. 
+    In other words, Open Source professionals have made their career supported in specific tooling and associated practices they have chosen and mastered.
+
+It is easy to spot the real skilled people with knowledge about it, I saw this also on my job. People thrust me about specific topics because I know how they works not because I read a tutorial but because I studied the internal stuff and experienced in a lot of different ways. Just create a repo for [hacktoberfest](https://hacktoberfest.digitalocean.com/) to do 4 pull requests and get a t-shirt cannot compare to with people that does real effective contribution and improve their skills on high level.  
```

Comment:
Do people [thrust](https://www.thefreedictionary.com/thrust) or trust you about specific topics??

Reply:
> I know how they work